codelists/tests/test_codeset.py

```diff
@@ -18,56 +19,56 @@
 # g   h   i   j
 
 
-def test_from_codes(subtests):
+def id_fn(value):
+    return value["description"]
+
+
+@pytest.mark.parametrize("example", examples, ids=id_fn)
```

Comment:
This is only used in this module as a data structure indexer. Maybe it could be an anonymous function (even though you use it a few... times), or a _pseudoprivate function. If the function continues to exist I'm not sure how to interpret the function or parameter name, can they be improved?

Maybe either:

```suggestion
@pytest.mark.parametrize("example", examples, ids=lambda example: example["description"]))
```

or

```suggestion
def _get_description(example):
    return example["description"]


@pytest.mark.parametrize("example", examples, ids=_get_description)
```

Reply:
I actually think I used a more descriptive function when I started, then copied the pytest docs for `id_fn`. (That [documentation actually uses `idfn`](https://github.com/pytest-dev/pytest/blob/6530f42e976647f0a78ce5aa7418017323616b3b/doc/en/example/parametrize.rst#L120-L123).)

So I've gone for the named function over a `lambda`.